### PR TITLE
Implement i64 and u64 uniform types

### PR DESCRIPTION
**Pull request by nicokoch**
cc #1157 
I'm not really sure if additional work is required to support 64 Bit integer attributes.
Also you should probably run the tests locally once to make sure it really works, since my drivers do not support this feature yet and thus the tests return early.

### CODE REVIEW

File: src/program/uniforms_storage.rs [hunk 1]
```diff
@@ -57,13 +57,25 @@ impl UniformsStorage {
...
+            )
+        );
+
+        macro_rules! uniform_i64(
+            ($ctxt:expr, $uniform:ident, $($params:expr),+) => (
+                unsafe {
+                    if $ctxt.extensions.gl_arb_gpu_shader_int64 {
+                        $ctxt.gl.$uniform($($params),+)
+                    } else {
+                        panic!("Double precision integers are not supported on this system.")
```
Comment (tomaka) [inline src/program/uniforms_storage.rs]: `Double precision integers` doesn't really make sense.

**Comment by tomaka** [comment]
Arg, my local rustc triggers a stack overflow.

**Comment by tomaka** [comment]
My driver doesn't support `GL_ARB_gpu_shader_int64`.

**Comment by tomaka** [comment]
I fixed the stack overflow in #1252

**Comment by nicokoch** [comment]
Very nice! That is a dirty workaround though :dancers:

**Pull request merged**